meta-mender-commercial/recipes-mender/mender-monitor/mender-monitor_git.bb

```diff
@@ -36,8 +41,7 @@ def mender_monitor_version_from_preferred_version(d, srcrev):
         # final tags, which will need their own recipe.
         return pref_version
     else:
-        # Else return "master-git${SRCREV}".
-        return "master-git%s" % srcrev
+        return "%s" % srcrev
```

Comment:
This won't anymore match [what `mender-qa` sets for CI builds](https://github.com/mendersoftware/mender-qa/blob/master/scripts/yocto-build-and-test.sh#L129). The trick is to have a known prefix that we can use from CI.

Right now it "works" because we don't have more recipes, but once 1.0.0 is out we still need a way from CI to prefer the master one